static/src/javascripts/projects/commercial/modules/commercial-features.js

```diff
@@ -72,18 +81,21 @@ class CommercialFeatures {
         this.videoPreRolls = this.dfpAdvertising;
 
         this.highMerch =
-            this.dfpAdvertising &&
+            (this.dfpAdvertising || this.adFree) &&
             !isMinuteArticle &&
             !isHosted &&
             !isInteractive &&
             !config.page.isFront &&
             !newRecipeDesign;
 
-        this.thirdPartyTags = externalAdvertising && !isIdentityPage;
+        this.thirdPartyTags =
+            (externalAdvertising || this.adFree) && !isIdentityPage;
 
         this.outbrain =
-            this.dfpAdvertising &&
+            (this.dfpAdvertising || this.adFree) &&
```

Comment:
OB is still a feature on ad-free. It's dependent on `dfpAdvertising` too, so this just makes sure it runs for ad-free.